### PR TITLE
yetris 2.3.0 (new formula)

### PR DESCRIPTION
**Pull request by lexi-lambda**
[yetris](https://github.com/alexdantas/yetris) is a Tetris implementation that runs in the terminal.

### CODE REVIEW

File: yetris.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Yetris < Formula
+  desc "Customizable Tetris(tm) for the terminal."
```
Comment (bfontaine) [inline yetris.rb]: You can remove this `(tm)` from the description.

File: yetris.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://github.com/alexdantas/yetris/archive/v2.3.0.tar.gz"
+  sha256 "720c222325361e855e2dcfec34f8f0ae61dd418867a87f7af03c9a59d723b919"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    version_str = shell_output("#{bin}/yetris --version").lines.first
+    assert_equal "v#{version}", version_str[/v\d+\.\d+\.\d+/]
```
Comment (bfontaine) [inline yetris.rb]: You can shorten this: `assert_match "v#{version}", shell_output("#{bin}/yetris --version")`.

**Comment by lexi-lambda** [comment]
@bfontaine Thanks for the comments; I've updated this with your suggested changes.

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

**Pull request closed without merging**